package.json

```diff
@@ -15,73 +15,80 @@
     "test:ember": "ember test",
     "test:ember-compatibility": "ember try:each"
   },
   "dependencies": {
+    "@babel/core": "^7.25.2",
     "@embroider/macros": "^1.0.0",
-    "ember-cli-babel": "^7.26.11",
+    "ember-cli-babel": "^8.2.0",
     "ember-modifier-manager-polyfill": "^1.2.0"
   },
   "devDependencies": {
-    "@ember/optional-features": "^2.0.0",
+    "@babel/eslint-parser": "^7.25.1",
+    "@babel/plugin-proposal-decorators": "^7.24.7",
+    "@ember/optional-features": "^2.1.0",
     "@ember/string": "^3.0.1",
-    "@ember/test-helpers": "^2.6.0",
-    "@embroider/test-setup": "^1.0.0",
+    "@ember/test-helpers": "^3.3.1",
+    "@embroider/test-setup": "^4.0.0",
     "@embroider/util": "^1.0.0",
-    "@glimmer/component": "^1.0.4",
-    "@glimmer/tracking": "^1.0.4",
+    "@glimmer/component": "^1.1.2",
+    "@glimmer/tracking": "^1.1.2",
     "@glint/template": "^1.0.2",
-    "babel-eslint": "^10.1.0",
-    "ember-auto-import": "^2.2.4",
-    "ember-cli": "~4.1.0",
-    "ember-cli-dependency-checker": "^3.2.0",
-    "ember-cli-htmlbars": "^6.0.1",
+    "concurrently": "^8.2.2",
+    "ember-auto-import": "^2.8.1",
+    "ember-cli": "~5.12.0",
+    "ember-cli-clean-css": "^3.0.0",
+    "ember-cli-dependency-checker": "^3.3.2",
+    "ember-cli-htmlbars": "^6.3.0",
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-terser": "^4.0.2",
-    "ember-disable-prototype-extensions": "^1.1.3",
     "ember-load-initializers": "^2.1.2",
-    "ember-page-title": "^7.0.0",
-    "ember-qunit": "^5.1.5",
-    "ember-resolver": "^8.0.3",
-    "ember-source": "~4.1.0",
+    "ember-page-title": "^8.2.3",
+    "ember-qunit": "^8.1.0",
+    "ember-resolver": "^12.0.1",
+    "ember-source": "~5.12.0",
     "ember-source-channel-url": "^3.0.0",
-    "ember-template-lint": "^3.15.0",
-    "ember-try": "^2.0.0",
-    "eslint": "^7.32.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-ember": "^10.5.8",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-qunit": "^7.2.0",
+    "ember-template-lint": "^6.0.0",
+    "ember-try": "^3.0.0",
+    "eslint": "^8.57.1",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-ember": "^12.2.1",
+    "eslint-plugin-n": "^16.6.2",
+    "eslint-plugin-prettier": "^5.2.1",
+    "eslint-plugin-qunit": "^8.1.2",
     "loader.js": "^4.7.0",
-    "npm-run-all": "^4.1.5",
-    "prettier": "^2.5.1",
-    "qunit": "^2.17.2",
-    "qunit-dom": "^2.0.0",
+    "prettier": "^3.3.3",
+    "qunit": "^2.22.0",
+    "qunit-dom": "^3.2.1",
     "release-it": "^14.11.6",
     "release-it-lerna-changelog": "^3.1.0",
-    "webpack": "^5.65.0"
+    "stylelint": "^15.11.0",
+    "stylelint-config-standard": "^34.0.0",
+    "stylelint-prettier": "^4.1.0",
+    "webpack": "^5.95.0"
   },
   "peerDependencies": {
-    "ember-source": "^3.8 || >= 4.0.0",
-    "@glint/template": "^1.0.2"
+    "@glint/template": "^1.0.2",
+    "ember-source": ">= 4.0.0"
```

Comment:
This adds support for Ember v6. Would be great if we could release it.

If you don't want to do another major yet, we could backport that small change of the supported `ember-source` peer dependency. But it would be great if we could make this addon compatible with Ember v6.